sdk/eidos_sdk/cpu/logic_unit.py

```diff
@@ -23,7 +23,7 @@ class LLMToolWrapper:
     eidos_handler: EidosHandler
     input_model: typing.Type[BaseModel]
 
-    async def execute(self, call_context: CallContext, tool_call: ToolCall) -> AsyncIterator[BaseStreamEvent]:
+    async def execute(self, tool_call: ToolCall) -> AsyncIterator[BaseStreamEvent]:
```

Comment:
ripped out call_context here since it was unused (and the wrong thing was being passed in)